install_script.sh.template

```diff
@@ -1144,7 +1173,7 @@ fi
 
 service_cmd="service"
 if [ "$SUSE11" == "yes" ]; then
-  service_cmd=`$sudo_cmd which service`
+  service_cmd=`$sudo_cmd which service || echo "service"`
```

Comment:
It seems you are by-passing a possible error here (probably to deal with your old opensuse), could you add a comment to explain why?

Reply:
Yeah good idea 👍 will add a comment about that (TLDR; on the SLES 11 ``service`` exist but since we can't legally use an SLES 11 image because the system isn't free we're testing on a opensuse 13.2 with bash downgraded but marked as on SUSE11 we need to put "service" as the default value if the ``which`` fails like it will in SUSE 13.2)